test/lib/utils/joinHelper.ts

```diff
@@ -0,0 +1,74 @@
+import { replaceWrapped, Token } from '../../../src';
+import { sendTransactionGetBalances } from './helper';
+import { expect } from 'vitest';
+import { JoinTxInput } from './types';
+
+export const doJoin = async (txInput: JoinTxInput) => {
```

Comment:
Nice to reuse this 👏
Could you add a bit of documentation to the function that explains what its doing/testing so we don't have to read through it all when we need a reminder.